plaidml/edsl/tests/edsl_test.cc

```diff
@@ -55,6 +55,20 @@ Tensor Softmax(const Tensor& X) {
   return E / N;
 }
 
+Tensor ConstAdd() {
+  std::string DEFAULT_DEVICE = "llvm_cpu.0";
```

Comment:
We should drop this, why is it needed?

Reply:
I see these two constructors for `Buffer`:
* `Buffer(const std::string& device, const TensorShape& shape)`
* `Buffer(plaidml_buffer* ptr, const TensorShape& shape)`

The `DEFAULT_DEVICE` string exists because I was using the first constructor, should I use the second constructor instead?

Reply:
This is used to choose which device to create the `Buffer` on. I've modified the code so that it grabs the current device name instead of using a hardcoded device name.